evolution/genome.py

```diff
@@ -9,7 +9,7 @@ class Genome:
     """
     _genome_ID = 0
 
-    def __init__(self, connections, input_size, output_size):
+    def __init__(self, connections=None, input_size=None, output_size=None, genome_to_clone=None):
```

Comment:
Jeśli `connections` nie jest `None`, ale `input_size` bądź `output_size` tak, to prowadzi to do niełapanego błędu.